osquery/distributed/distributed.cpp

```diff
@@ -117,11 +119,31 @@ Status Distributed::serializeResults(std::string& json) {
     doc.add(result.request.id, arr, queries_obj);
     doc.add(result.request.id, result.status.getCode(), statuses_obj);
     doc.add(result.request.id, result.message, messages_obj);
+
+    auto obj = doc.getObject();
+    if (performance_.count(result.request.id) > 0) {
+      auto perf = performance_[result.request.id];
+      obj.AddMember("wall_time_ms",
+                    static_cast<uint64_t>(perf.wall_time_ms),
+                    obj.GetAllocator());
+      obj.AddMember("user_time",
+                    static_cast<uint64_t>(perf.user_time),
+                    obj.GetAllocator());
+      obj.AddMember("system_time",
+                    static_cast<uint64_t>(perf.system_time),
+                    obj.GetAllocator());
+      obj.AddMember("last_memory",
```

Comment:
Unnecessary `last` still there?
```suggestion
      obj.AddMember("memory",
```